style.css

```
@@ -1,6 +1,8 @@
 .wp-block-code {
 	border: 0;
 	padding: 0;
+	-webkit-text-size-adjust: none;
+	text-size-adjust: none;
```

I am also not familiar with this property. Given that `normalize.css` uses `100%` and that this is also [used in Twenty Twenty-One](https://github.com/WordPress/wordpress-develop/blob/5334a8dcca2704e800e11419de9c052db24d9ba8/src/wp-content/themes/twentytwentyone/style.css#L357), I think we should use it instead of `none`.

```suggestion
	-webkit-text-size-adjust: 100%;
	text-size-adjust: 100%;
```